pylossless/pipeline.py

```diff
@@ -1268,7 +1363,15 @@ def _run(self):
 
             # OPTIONAL: Flag chs/epochs based off fixed std threshold of time axis
             self.flag_epochs_fixed_threshold(picks=picks)
-            self.flag_channels_fixed_threshold(picks=picks)
+            if "flag_channels_fixed_threshold" in self.config:
+                if "threshold" in self.config["flag_channels_fixed_threshold"]:
+                    thresh = self.config["flag_channels_fixed_threshold"]["threshold"]
+                else:
+                    thresh = 5e-5
+                msg = "Flagging Channels by fixed threshold"
+                self.flag_channels_fixed_threshold(
+                    threshold=thresh, picks=picks, message=msg
+                    )
```

Comment:
```suggestion
            if "flag_channels_fixed_threshold" in self.config:
                msg = "Flagging Channels by fixed threshold"
                kwargs = dict(picks=picks, message=msg)
                if "threshold" in self.config["flag_channels_fixed_threshold"]:
                    kwargs["threshold"] = self.config["flag_channels_fixed_threshold"]["threshold"]
                self.flag_channels_fixed_threshold(**kwargs)
```

Let the default value of `self.flag_channels_fixed_threshold(...)` assign this 5e-5 value so that we don't hardcode at two places the same default value.